pkgs/applications/graphics/krita/wrapper.nix

```diff
@@ -0,0 +1,28 @@
+{ libsForQt5
+, symlinkJoin
+, makeWrapper
+, unwrapped ? libsForQt5.callPackage ./. { }
+, krita-plugin-gmic
```

Comment:
`krita-plugin-gmic` is already passed in as `plugins` below, and it is not used. Maybe you forgot to remove this line?

Reply:
```
warning: Git tree '/home/lelgenio/.local/mount/bigboy/projects/git/nixpkgs' is dirty
error:
       error: undefined variable 'krita-plugin-gmic'

       at /nix/store/qij6g3w0jp4cvqzavwdrz1lh3mc30rcj-source/pkgs/applications/graphics/krita/wrapper.nix:6:5:

            5| , plugins ? [
            6|     krita-plugin-gmic
             |     ^
            7|   ]
```

AttrSet de-structuring is recursive, the value in the plugins array **is** the one from the argument.

```nix
nix-repl> ({ a, b ? a }: b) { a = "A"; }
"A"
```

Reply:
Oh I see!